Assets/Scripts/Level/TutorialTriggerGroup.cs

```diff
@@ -61,5 +62,7 @@ public void ResetTriggers()
         {
             trigger.ResetTrigger();
         }
+        string[] blankString = { };
+        gameUI.SetupTutorialTexts(blankString);

```

Comment:
could just be `gameUI.SetupTutorialTexts(new string[] {});`

Reply:
Good point, was messing with declarations a bit to see whether I needed "" or {} and forgot to update.